test/test_nn.py

```diff
@@ -9382,6 +9382,18 @@ def test_GroupNorm_empty(self, device):
             with torch.backends.cudnn.flags(enabled=False):
                 self._test_module_empty_input(mod, inp)
 
+    def test_ReplicationPad_empty(self, device):
+        for mod, inp in [
+                (torch.nn.ReplicationPad1d(3), torch.randn(0, 3, 10)),
+                (torch.nn.ReplicationPad2d(3), torch.randn(0, 3, 10, 10)),
+                (torch.nn.ReplicationPad3d(3), torch.randn(0, 3, 10, 10, 10))]:
+            mod = mod.to(device)
```

Comment:
nit: replication pad does not have parameters, so you don't have to move it to device